olp-cpp-sdk-dataservice-read/tests/PrefetchRepositoryTest.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include <gtest/gtest.h>
+
+#include <repositories/PrefetchTilesRepository.h>
+
+namespace {
+using namespace olp::dataservice::read;
+
+class PrefetchRepositoryTest : public ::testing::Test,
```

Comment:
Not necessary. Seems most of the method are static either way so just use without fixture.
For this you can easily create a testable:

```cpp
class PrefetchRepositoryTestable : public repository::PrefetchTilesRepository {
  // Expose protected methods to public scope
  using repository::PrefetchTilesRepository::GetSubQuads;
};

TEST(PrefetchRepositoryTest, SplitTreeLevel5) {
  PrefetchRepositoryTestable repository;
  ....
}
```